api/common/v1alpha1/match.go

```diff
@@ -0,0 +1,16 @@
+package v1alpha1
+
+type Match struct {
+	// SpiffeId defines a matcher configuration for SpiffeId matching
+	SpiffeId *SpiffeIdMatch `json:"spiffeId,omitempty"`
+}
+
+// +kubebuilder:validation:Enum=Exact;PathPrefix
```

Comment:
Should it be `PathPrefix` or maybe just `Prefix`?

Reply:
that was in the issue, but I checked madr again and it should be `Path` updated